Source/WebKit/Platform/IPC/Connection.h

```diff
@@ -119,6 +119,7 @@ enum class Error : uint8_t {
     FailedToAcquireReplyBufferSpan,
     StreamConnectionEncodingError,
 };
+using Error = Markable<ErrorType, EnumMarkableTraits<ErrorType, 0>>;
```

Comment:
I was confused that something named `Error` still had a happy value!
I'd personally rename this `SuccessOrError` or `MaybeError` or similar, to show that it's not *only* an error.

Reply:
And if you did that, then you could rename `ErrorType` back to just `Error`, which would be correct, and remove a lot of changes in this patch. 😉

Reply:
Renaming `Error` to something else required a lot more changes than renaming `Error` to `ErrorType` and `using Error = ...`. But I agree this is a good follow up refactoring.

There's not a happy value; `NoError` is gone. (unless I miss what you mean by "happy value")

Reply:
> There's not a happy value; `NoError` is gone. (unless I miss what you mean by "happy value")

The empty case of `Markable` is the "happy" value I'm thinking of. So we have a type named `Error` that could still have a null/empty non-value meaning "actually there's no error here"; that's why I'm suggesting a name that reflects this possibility of no-error.

Reply:
(But fine as a follow-up, since at least now the no-error case cannot be silently handled like a real error.)